src/screens/OtherProfile/hooks/useOtherProfileScreenHooks.tsx

```diff
@@ -34,7 +34,7 @@ const useOtherProfileScreenHooks = (targetUserProfileId: string, username: strin
   const [, setOtherProfileUserId] = React.useState<string>('');
   const [other_id, setOtherId] = React.useState('');
   const [isCurrentFollowed, setIsCurrentFollowed] = React.useState(
-    getIsMeFollowingTargetStatus(params.data.other_id, params?.data?.username) ||
+    getIsMeFollowingTargetStatus(params?.data?.other_id, params?.data?.username) ||
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good one. It adds optional chaining to `params?.data?.other_id` which will prevent potential runtime errors if `params` or `params.data` are undefined. This is an improvement in error handling and robustness of the code.
```diff
- getIsMeFollowingTargetStatus(params.data.other_id, params?.data?.username) ||
+ getIsMeFollowingTargetStatus(params?.data?.other_id, params?.data?.username) ||
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->